airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -197,20 +197,11 @@
     }
 
     function createMediaElementPlayer() {
-
-        var player = $('#audioPlayer').get(0);
+        // Manually run MediaElement.js initialization.
```

Comment:
It might be worth providing a link to this pullrequest in the comment, to explain why we're manually initializing MEjs :)

Reply:
Done!

Reply:
Done!